twyla/chat/templates/quickreplies.py

```diff
@@ -15,10 +15,11 @@ class QuickReplies:
     quick_replies: List[QuickReply] = field(default_factory=list)
     template_type: str = "fbmessenger.quick_reply"
 
-    def add(self, title: str) -> None:
+    def add(self, title: str, payload: str) -> None:
```

Comment:
We should probably move this to accept `QuickReply` type as a parameter like with the other templates just for consistency atleast.

Reply:
true, will change that